### PR TITLE
[update][l] Updating the code

### PR DESCRIPTION
**Pull request by gradedSystem**
Updating the scripts with parsing `.xls` file:
- added function to update `datapackage.json`'s `last-updated` and `version` fields
- Updated the functionality of the code

cc @anuveyatsu

### CODE REVIEW

File: scripts/process.py [hunk 1]
```diff
@@ -1,33 +1,85 @@
...
+import xlrd
+import requests
+
+from datetime import datetime, timedelta
 
 cache = 'cache'
-if not os.path.exists(cache):
-    os.makedirs(cache)
+data = '../data/gdp.csv'
+url = 'https://api.worldbank.org/v2/en/indicator/NY.GDP.MKTP.CD?downloadformat=excel'
```
Comment (anuveyatsu) [inline scripts/process.py:11]: @gradedSystem I don't think getting an excel version is a good idea. It only adds complexity. Can you check this landing page for the GDP data from WorldBank: https://data.worldbank.org/indicator/NY.GDP.MKTP.CD 

You can find API for CSV format there: https://api.worldbank.org/v2/en/indicator/NY.GDP.MKTP.CD?downloadformat=csv

File: scripts/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+requests==2.32.3
+xlrd==2.0.1
```
Comment (anuveyatsu) [inline scripts/requirements.txt:2]: looks like we don't really need xlrd.

File: scripts/process.py (hunk 1 above)
Comment (gradedSystem) [reply scripts/process.py]: @anuveyatsu I changed the code and with the format=csv

File: scripts/process.py [hunk 3]
```diff
@@ -1,39 +1,60 @@
 import os
+import io
 import csv
 import json
-import xlrd
```
Comment (anuveyatsu) [inline scripts/process.py]: @gradedSystem should we remove this from requirements.txt now?

Comment (gradedSystem) [reply scripts/process.py]: @anuveyatsu correct will remove that right now

Comment (gradedSystem) [reply scripts/process.py]: @anuveyatsu fixed

**Pull request merged**